### DISCUSSION TITLE
A website for a collection of HTA packages - HTAverse?

### DISCUSSION THREAD

**Discussion by n8thangreen** [HTAhackathon2024/.github#3] [Ideas]
Create something (via GitHub pages) that is an HTA version of something like

[pharmaverse](https://pharmaverse.org/)

 or

[tidymodels.](https://www.tidymodels.org/packages/)

**Comment by n8thangreen** [comment]
I did start a very basic page for this ages ago!
https://n8thangreen.github.io/health_economics_R_packages/

**Reply by flossybap** [reply]
Oh this is cool! R for HTA have a list of ones. ZIN have packages they approve also.

**Comment by bisacciamd** [comment]
This would be very useful and interesting to do. 

Somebody in Zoom earlier today mentioned [SHEPRD](https://hermes-sheprd.netlify.app), developed during a previous hackathon.

We could work on this idea and make it a standalone Github Pages website in Quarto. I would love to contribute to this - I have expertise in building Quarto websites, but little knowledge of R packages for HTA.

**Comment by n8thangreen** [comment]
Heres a table of packages curated by @RayHendy 
[Open source packages.xlsx](https://github.com/user-attachments/files/16265318/Open.source.packages.xlsx)

**Reply by CiaranEONeill** [reply]
As someone who is coming new to R (fool that I am) this is very useful to see examples

**Reply by alex-vand** [reply]
Yeah, this is great! Partly related to this, I had an idea a while back to create a 'GitHub native' inventory resource for open-source health economic models - Originally, I thought to focus only on model / health economic analysis implementations vs. the underlying 'tooling' packages, but this could also extend to package vingette examples and the instructional resources included here. The output would be v similar to the above but perhaps available on a website or via a Shiny App.

In case there is interest in exploring something along these lines for this project, these were my half-baked thoughts for an implementation:

1. Create a metadata file template for the model developer / GitHub maintainer to complete and add to the relevant repo. This would include similar info to what you have above @RayHendy and could be made available as a form to allow for easy field value controls (instead of managing that later down the line). Either way, the template or form should be downloadable so that it can easily be added to the associated repo.

2. Write a GitHub API query to identify all (public) repos with the metadata file and extract file or data directly.

3. Combine all data in to a master dataset.

4. Make available in a filterable/ searchable format on a website or hosted Shiny App.

**Comment by n8thangreen** [comment]
| Available | Platform | Repository | Title                                                                 | Published | Journal                                | DOI                           | Author               | Intervention                  | Disease               | Type                  | Category           | Country      | Income Level          | Model                                    | Technique     | Software | License         | Access     | Funder      | Outcomes                                  | Date         | URL                                                                 |
|-----------|----------|------------|----------------------------------------------------------------------|-----------|----------------------------------------|-------------------------------|----------------------|------------------------------|-----------------------|-----------------------|-------------------|--------------|----------------------|-----------------------------------------|---------------|----------|----------------|------------|-------------|------------------------------------------|--------------|--------------------------------------------------------------------|
| Yes       | Github   |            | Extended cost-effectiveness analysis of school-based HPV vaccination rollout in Ethiopia | No        | Not applicable                         | Not applicable                 | Emily Reichart       | Vaccination                  | HPV                   | Infectious disease    | Infectious disease | Ethiopia     | Low income           | Simulation                               | Simulation    | R        | Not described   | Open source| Unspecified | Not reported                             | 24-May-22    | [GitHub](https://github.com/emreichert13/HPVVaxxinEthiopia)         |
| Yes       | Github   |            | Finding Early Stage HIV Infections in a Hot Epidemic: A Cost-Effectiveness Analysis of The Sabes Study in Lima, Peru | Yes       | Lancet Regional Health - Americas      | 10.1016/j.lana.2022.100281    | Blythe Adamson       | Sabes                        | HIV                   | Infectious disease    | Infectious disease | Peru         | Upper middle income  | Markov model                             | Markov        | R        | CC-BY-NC-ND-4.0 | Open source| Government  | Costs, LYs, QALYs and ICER                | 25-Nov-19    | [GitHub](https://github.com/blythejane/lima-hiv-testing)             |
| Yes       | GitLab   |            | Fine-grained mathematical modeling for cost-effectiveness evaluation of public health policies for cervical cancer, with application to a Colombian case study | Yes       | BMC Public Health                      | 10.1186/s12889-023-16022-x    | Daniela Angulo       | Vaccination                  | Cervical Cancer       | Oncology              | Oncology           | Colombia     | Upper middle income  | Deterministic continuous-time epidemics model | Simulation    | R        | CC-BY-4.0       | Open Source| Unspecified | Costs, DALYs, and ICER                    | 25-Apr-23    | [GitLab](https://gitlab.oit.duke.edu/im90/cervicalcancer)           |
| Yes       | Github   | aldvmm     | Fitting Adjusted Limited Dependent Variable Mixture Models to EQ-5D   | Yes       | The Stata Journal                       | 10.1177/1536867X1501500       | Mark Pletscher       | Not reported                 | Not reported          | Instructional         | Modelling tools    | Not reported | Not reported         | Health state utilities                   | Health utility| R        | Not described   | Open source| Government  | Not reported                             | 11-Oct-22    | [GitHub](https://github.com/pletschm/aldvmm)                        |
| Yes       | Zenodo   | McMarcel   | Health and economic burden of Respiratory Syncytial Virus (RSV) disease and the cost-effectiveness of potential interventions against RSV among children under 5 years in 72 Gavi-eligible countries | Yes       | BMC Medicine                           | 10.1186/s12916-020-01537-6    | Xiao Li              | Vaccination                  | RSV                   | Infectious disease    | Infectious disease | Low income countries | Low income           | McMarcel model                            | Markov        | R        | CC-BY-4.0       | Open source| Government  | Mortality and DALYs                       | 12-Feb-20    | [Zenodo](https://zenodo.org/record/3663447)                         |
| Yes       | Github   | hedtree    | Hedtree: perform cost-effectiveness analyses using decision tree modelling | No        | Not applicable                         | Not applicable                 | Nathan Green         | Not reported                 | Not reported          | Instructional         | Modelling tools    | Not reported | Not reported         | Decision tree model                       | Decision tree | R        | CC-BY-4.0       | Open source| Unspecified | Not reported                             | 01-Dec-22    | [GitHub](https://github.com/petedodd/Hedtree)                       |
| Yes       | Github   | heemod     | heemod: Markov Models for Health Economic Evaluations                 | Yes       | Value in Health                         | 10.1016/j.jval.2016.09.133    | Kevin Zarca          | Vaccination                  | Influenza             | Infectious disease    | Infectious disease | UK           | High income          | semi-Markov and non-homogeneous Markov models | Markov        | R        | GPL-3.0         | Open source| Academic    | Costs, QALYs, and ICER                    | 22-Mar-23    | [GitHub](https://

**Reply by n8thangreen** [reply]
part of the xl table which chatgpt converted for me!

**Comment by bisacciamd** [comment]
That's a true goldmine... Bravo @RayHendy

If I think of the website's structure, here's how I would do it (similar to [tidymodels](https://www.tidymodels.org/start/)):

# The HTAveRse
- **Homepage**: welcomes visitors, presents the rationale for R in HTA, lists main content...
- **Get started**: presents key packages, procedures, and links to R&HTA initiatives
- **Discover the HTAveRse**: in a single page with a [sidebar menu](https://quarto.org/docs/websites/website-navigation.html#side-navigation), it has several paragraphs, each detailing one aspect (e.g. developing models, visualization, CEA, Value of information, etc.)
- **About**: how the website was developed (2024 hackathon, the people, etc.)

Then, if this goes on we can even think of
- having a **Contact** page so that developers/users can reach out to signal new/missing R packages.
- having a maintenance procedure so that we have this updated over time.

Thoughts?

**Reply by bisacciamd** [reply]
@n8thangreen should I / can I get the green light to start developing this in a repo in this organization?

Also, I am happy to lead or be lead on this as required. 😊

**Comment by carolinaheu** [comment]
This sounds like a great idea! Certainly very useful for colleagues just starting on their R in HTA journey :)

**Comment by alex-vand** [comment]
Within this, it could be good/helpful to categorise packages by their primary focus/ area e.g., survHE, flexsurv, ... : Survival analysis; hesim, heemod, ... : Health economic model implementation (and analysis) etc. [SHEPRD](https://hermes-sheprd.netlify.app/) mostly covers the latter. Topics covered by [NICE TSDs](https://www.sheffield.ac.uk/nice-dsu/tsds/full-list) (other HTA bodies are available) might be useful as a basis for a list of categories.

**Reply by mdallat** [reply]
I would like to join this group - I have some training in health economics but not so much on the technical side so knowing what's already built and ready to use (maybe with minor modifications to suit a particular project) would be very helpful for my work!!

**Comment by RayHendy** [comment]
Attempting to advance the flexsurvPLUS package but timelines are a little tight. If I don't make further progress I'd like to join this group?

**Comment by carolinaheu** [comment]
I would also be interested in joining this group as well!

**Comment by CiaranEONeill** [comment]
Nice to e-meet you Caroline and meet you in person next week.

bw,

Ciaran
________________________________

**Comment by yishu-lin** [comment]
This is interesting to me too. A comparison among similar packages might be useful.

**Comment by bijusoman** [comment]
I, too, would love to contribute to this initiative. I can contribute from a user perspective and from a public health point of view. I am familiar with R and Health Economics principles, but not an expert in either. I would like the repository to have links to selected resource materials for people who are interested to know more of the theoretical aspects as well.
